src/Umbraco.Web/PropertyEditors/ContentPickerPropertyEditor.cs

```diff
@@ -33,17 +33,12 @@ protected override PreValueEditor CreatePreValueEditor()
 
         internal class ContentPickerPreValueEditor : PreValueEditor
         {
-            [PreValueField("showOpenButton", "Show open button", "boolean")]
+            [PreValueField("showOpenButton", "Show open button (this feature is in preview!)", "boolean", Description = " Opens the node in a dialog")]

```

Comment:
OK @Shazwazza created this task here for future sprint/s, not a priority but a nice to have
http://issues.umbraco.org/issue/U4-9396